### DISCUSSION TITLE
requestSubscription

### DISCUSSION THREAD

**Discussion by avneesh74** [hyochan/react-native-iap#1329] [Q&A]
Hi there, Im getting issue on try requestSubscription, request completes successfully but it works like product purchase in sandbox testing it show only five minutes then expired, user have to subscribe his subscription from google play store ,

it was working before

it is showing 1 tap to pay earlier it was showing subscribe, and it was getting auto renewed, as per the subscription expiry date. 

i have also checked this for non test users but still i am get this only.

### Tested environment (Emulator? Real Device?) : Real Device

![Screenshot_20210510-113217__01](https://user-images.githubusercontent.com/44198692/117614818-7d3fcf80-b186-11eb-9faa-438fd5570ef0.jpeg)

**Comment by nirbhaisingh1313** [comment]
@avneesh74 found any solution ????

**Comment by hyochan** [comment]
This is very strange. Looks like this needs to be asked to Google or [stackoverflow](https://stackoverflow.com).

Hope you can share a link here so people can see either.

**Comment by avneesh74** [comment]
Hello Guys,

Google's policy has been changed regarding the subscription in INDIA. 

Please check below below post for more details

https://www.indiatoday.in/technology/news/story/google-is-pausing-free-trials-for-play-store-apps-in-india-blame-the-new-rbi-regulation-1798745-2021-05-04
